data/events.yml

```diff
@@ -2,17 +2,36 @@
 #
 # Available properties per event
 #  eventId: Should be in format "yyyymmdd-suffix". The yyyymmdd is the date
-#  meetupUrl: (optional) Link the event organization itself.
+#  eventUrl: (optional) Link the event organization itself.
+#  eventName: Name of the event
 #  location: Should be in format "city"
 #  date: Must be in format "yyyy-mm-dd". If multiple days cover our project, only note the first day
-#  talks: (optional) If the CFP is not yet accepted, do not include it yet.
-#    - title: Session title (without the conference or organizer name).
+#  talks: (optional)
+#    - title: Session title (without the event name).
 #      presenters: (optional) Comma separated list of presenters.
 #
 # Order events by date descending
 
-- eventId: 20221124-brussels
-  meetupUrl: https://www.meetup.com/belgian-java-user-group/events/289451067/
+- eventId: 20231002-devoxx
+  eventUrl: https://devoxx.be/
+  eventName: "Devoxx 2023"
+  location: Antwerp
+  date: 2023-10-02
+
+- eventId: 20230205-fosdem
+  eventUrl: https://foojay.io/today/friends-of-openjdk-at-fosdem-2023
```

Comment:
@GeertjanWielenga Is there a link directly to the Fosdem Java room schedule we could use?